docs/learn/learn-staking.md

```diff
@@ -9,189 +9,357 @@ slug: ../learn-staking
+production during a specific `era`. An era is a period of {{ polkadot: 24 hours :polkadot }}{{ kusama: 6 hours :kusama }} during
+which an `active set` of validators is producing blocks and performing other actions on chain. This
+means that not all validators are in the active set and such set changes between eras. Each era is
+divided into 6 epochs or `sessions` during which validators are assigned as block producers to
+specific time frames or `slots`. This means that validators know the slots when they will be
+required to produce a block within a specific session, but they do not know all the slots within a
+specific era. Having sessions adds a layer of security because it decreases the chance of having
+multiple validators assigned to a slot colluding to harm the network.
+
+### Staking Rewards
+
+Validators who produce a block are rewarded with tokens, and they can share rewards with their
+nominators. Both validators and nominators can stake their tokens on chain and receive staking
+rewards at the end of each era. The staking system pays out rewards equally to all validators
+regardless of stake. Thus, having more stake in a validator does not influence the amount of block rewards
+it receives. This avoids the centralization of power to a few validators. There is a probabilistic
+component in the calculation of rewards, so they may not be exactly equal for all validators. In
+fact, during each era validators can earn `era points` by doing different tasks on chain. The more the points, the higher the reward for a specific era. This promotes validators' activity on chain.
+To know more about era points, and how and on which basis they are distributed visit the
+[dedicated page](../maintain/maintain-guides-validator-payout.md). Distribution of the rewards is pro-rata to all stakers after the validator's commission is deducted.
+
+### Skin in the game when Staking
+
+The security of PoS networks depends on the amount of staked tokens. To successfully attack the network, a malicious actor would need to accrue a large number of tokens or would need different participants to collude and act maliciously. If there is an attack in the case of NPoS, both the validator(s) and nominators will be
+`slashed` resulting in their stake being partially or fully confiscated by the network and then deposited to the treasury. There is little interest for a rational network participant to act in a harmful way because NPoS ensures that all participants can be
+held accountable for their bad actions. In NPoS, validators are paid equal rewards regardless of the amount of stake backing them, thus avoiding large payouts to few large validators which might lead to centralization.
+
+## Being a nominator
+
+### Tasks and Responsibilities of a Nominator
+
+Since validator slots are limited, most of those who wish to stake their DOT and contribute to the
```

Comment:
DOT should be DOT / KSM?

Reply:
yep, thanks. This is fixed now.

Reply:
We can also conditionally render this doc for Polkadot and Kusama